src/Std/Time/Zoned/Database/TZdb.lean

```diff
@@ -89,4 +89,17 @@ def readRulesFromDisk (path : System.FilePath) (id : String) : IO ZoneRules := d
 
 instance : Std.Time.Database TZdb where
   getLocalZoneRules db := localRules db.localPath
-  getZoneRules db id := readRulesFromDisk db.zonesPath id
+
+  getZoneRules db id := do
+    let env ← IO.getEnv "TZDIR"
+
+    if let some path := env then
+      let result ← readRulesFromDisk path id
```

Comment:
Could you update the error message generated in `parseTZIfFromDisk` so that it contains the path that was tried?

Reply:
Added the path to the error message.